index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Great job @rozhi-98 ✋

- [ ] For the styles to reflect very well on the `Hello-Microverse` text, you should link your stylesheet with your `index.html` file. At the moment, the styles are not reflecting on the `h1`.